robot/Cumulus/resources/NPSP.robot

```diff
@@ -136,6 +136,27 @@ API Create GAU
     &{gau} =     Salesforce Get  ${ns}General_Accounting_Unit__c  ${gau_id}
     [return]         &{gau}  
 
+API Create GAU Allocation
```

Comment:
Can you please provide some documentation on the objective of this API call?

Reply:
done